setup/account_cryptocurrency/odoo/addons/account_cryptocurrency

```diff
@@ -0,0 +1 @@
+../../../../account_cryptocurrency
```

Comment:
remove these files

Reply:
Why? This is a symlink and it is required to be in the setup folder for the pip install method.